### DISCUSSION TITLE
obfuscated script with —enable-rft does not run

### DISCUSSION THREAD

**Discussion by ddi98** [dashingsoft/pyarmor#1144] [Q&A]
As reported in the documentation I replaced all the import * with the names of the imported functions and this meant that the obfuscation of the code ended without errors. but when I start the app.py file it doesn't start.
Another strange thing the only option working so far --mix-str after replacing the import * with the names of the functions when I start the app.py file it returns an import error

_Originally posted by @ddi98 in https://github.com/dashingsoft/pyarmor/issues/1111#issuecomment-1501699472_

**Comment by jondy** [comment]
Please try pyarmor `8.1.1`

**Comment by ddi98** [comment]
if I obfuscate with --enable-rft it doesn't start the application

**Comment by ddi98** [comment]
Is there any news on solving the problem?  now I can only use —mix-str without any other protection scheme

**Comment by null-routed** [comment]
I am experiencing the same issue with RFT. After obfuscating the code with RFT, my script gets stuck before launching. I attempted to enable RFT_trace, but I'm unsure how to debug the code the trace provides. Because of this, I cannot even pinpoint the part of the code that's causing the problem. Can you please explain how to debug RFT-obfuscated code? @jondy

**Comment by jondy** [comment]
Here it is
https://pyarmor.readthedocs.io/en/latest/tutorial/advanced.html#using-rftmode

And 
https://pyarmor.readthedocs.io/en/latest/topic/rftmode.html

**Comment by ddi98** [comment]
@jondy the problem for me is not only --enable-rft but also --enable-jit --assert-call etc
